tools/c7n_mailer/c7n_mailer/deploy.py

```diff
@@ -28,11 +28,11 @@
 
 logger = logging.getLogger('custodian.mailer')
 log_format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
-logging.basicConfig(level=logging.DEBUG, format=log_format)
+logging.basicConfig(level=logging.INFO, format=log_format)
```

Comment:
default things to INFO, you can decide to run --debug from CLI, or as well you have the option of debug: true in your mailer.yml